test/chunking-form/samples/amd-id-code-split/_config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
Your test names are switched: The code-split test is not doing the code splitting while the other one does